content/applications/general/auth/azure.rst

```diff
@@ -2,9 +2,158 @@
 Microsoft Azure sign-in authentication
 ======================================
 
-Due to specific requirements in Azure's OAuth implementation, Microsoft Azure OAuth identification
-is NOT compatible with Odoo at the moment.
+The Microsoft Azure OAuth sign-in authentication is a useful function that allows Odoo users to sign
+in to their database with their Microsoft Azure account.
+
+This is particularly helpful if the organization uses Azure Workspace, and wants employees within
+the organization to connect to Odoo using their Microsoft Accounts.
+
+.. warning::
+   Databases hosted on Odoo.com should not use Oauth login for the owner or administrator of the
+   database as it would unlink the database from their Odoo.com account. If Oauth is set up for that
+   user, the database will no longer be able to be duplicated, renamed or otherwise managed from
+   the Odoo.com portal.
+
 
 .. seealso::
    - :doc:`../../productivity/calendar/outlook`
    - :doc:`/administration/maintain/azure_oauth`
+
+Configuration
+=============
+
+The integration of the Microsoft sign-in function requires configuration both on Microsoft and Odoo.
+
+Odoo System Parameter
+---------------------
+
+First activate the :ref:`developer mode <developer-mode>`, and then go to :menuselection:`Settings
+--> Technical --> System Parameters`.
+
+Add in the following system parameter: `auth_oauth.authorization_header` and set the value to
+`1`.
```

Comment:
Be thorough please. It's not apparent how to do this if you've never done it before.

```suggestion
Click :guilabel:`Create` and on the new/blank form that appears, add in the following system
parameter `auth_oauth.authorization_header` to the :guilabel:`Key` field, and set the
:guilabel:`Value` to `1`, as well. Then click :guilabel:`Save` to finish.
```